admin/resources/js/components/poolCandidate/CreatePoolCandidate.tsx

```diff
@@ -140,10 +146,14 @@ export const CreatePoolCandidateForm: React.FunctionComponent<CreatePoolCandidat
               id="pool"
               label="Pool: "
               name="pool"
-              options={[
-                { value: "", label: "Select a pool...", disabled: true },
-                ...poolOptions,
-              ]}
+              options={
+                poolOptionsSingleOption
+                  ? [poolOptionsSingleOption]
+                  : [
+                      { value: "", label: "Select a pool...", disabled: true },
+                      ...poolOptions,
+                    ]
+              }
```

Comment:
Instead of doing this, you can set react-hook-forms `defaultValues` object so that the pool field is automatically set. This is done in the useForm hook (on line 73):

 `const methods = useForm<FormValues>({ defaultValues: { pool: poolId } });`. 

 That way all you need to do is add the `disabled` prop to the pool form field, in this case the Select component.

Reply:
I guess my thought with `poolOptionsSingleOption` and then the ternary expression for the `options` value was to allow for the possibility that the create pool candidate form might be linked to somewhere where the `poolId` is not set. Otherwise, why do we need to use a select input?